packages/discovery-provider/src/tasks/update_aggregates.py

```diff
@@ -201,6 +201,31 @@
   group by
     owner_id
 ),
+genre_counts as (
+  select
+    owner_id as user_id,
+    genre,
+    count(*) as count
+  from
+    tracks t
+  where
+    t.is_current is true
```

Comment:
nooooooooo

Reply:
hahahahaha

Reply:
will leave this for a future PR ... because they all do this filter still 👿